main.go

```diff
@@ -47,8 +47,12 @@ var exitFirst = flag.Bool("x", false, "exit on the first failure")
 var errorShown = false
 var usage func(string)
 
-func displayUnknown(name string) {
-	fmt.Printf("%s ?\n", name)
+func displayUnknown(tmpl *template.Template, name string, ref *retrodep.Reference, root string) {
+	if ref == nil || *templateArg != "" {
+		fmt.Printf("%s%s ?\n", name, root)
```

Comment:
I think this can just be `"%s ?\n", name` if name is the name of the package.

Reply:
No, it cannot. `topLevelMarker` and `projectRoot` must be distinguished to be able to call `display()` correctly, and also for compatiblity. Please note, that the `projectRoot` is there for a case when the `ref` is nil.